README.md

```diff
@@ -54,6 +54,8 @@ This module will install packages, create configuration and start services neces
 
 Plugin sync is required if the custom sensu types and providers are used.
 
+This module has a soft dependency on the [puppetlabs/apt](https://forge.puppet.com/puppetlabs/apt) module for systems using `apt`.
```

Comment:
Can you specify the version string like you would with `metadata.json`. I think it is `>= 4.0.0 > 5.0.0`